oran_o2ims_suite_test.go

```diff
@@ -0,0 +1,326 @@
+package main_test
+
+import (
+	"bytes"
+	"os"
+	"testing"
+
+	"io"
+	"net/http"
+  "crypto/tls"
```

Comment:
Looks like this code hasn't been automatically formatted. Try to do so: `make fmt`